# Status message for Attitude ESKF
Header header

# Magnetometer status
#  0 = Idle (or disabled), unused in estimation
#  1 = Running calibration
#  2 = Calibrated, estimation is magnetically referenced
int32 magStatus
